index.html

```diff
@@ -506,7 +506,7 @@ <h4 class="desktop-only">Uber</h4>
 
             <div id="imagen"><img src="media/escritoriotrabajo4.png" alt="" class="escritorio-trabajo-foto-izquierda"></div>
          
-         </div>
+         </main>
```

Comment:
- I recommend that you don't forget to change your </main> tag to </section> to follow good HTML practices.